### ISSUE TITLE
Would like to view multiple projects in the editor.

### ISSUE THREAD

**Issue by davidoshiro** [defold/editor2-issues#2113]
I would love to see multiple projects in the editor. Working on multiple projects is cumbersome by having to close editor and re-open then choose new project. Also, reusing code and referencing code in other projects is very clumsy. At least allow for multiple instances of the editor to run at the same time.

Thank you.

### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2018-07-30T17:09:20.393386</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>6f9454d6599a1082438e69d68c4d3465b2d8ed47</td></tr>
<tr><td>Defold engine sha</td><td>b2ef3a19802728e76adf84d51d02e11d636791a3</td></tr>
<tr><td>Defold version</td><td>1.2.134</td></tr>
<tr><td>GPU</td><td>Intel HD Graphics 4000 OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 INTEL-10.36.19</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.13.6</td></tr>
<table>

**Comment by amel** [comment]
Are you referring to [this post](https://forum.defold.com/t/a-project-explorer-and-opening-projects-from-the-menu-bar/25098/3)?

**Comment by Jerakin** [comment]
You can have multiple editors open if you open one from the terminal `open /Applications/Defold.app --new`
You might get some more crashes and errors though (but not too many to not use it), remember that when reporting them to note if you had multiple editors open just in case it is important.

**Comment by davidoshiro** [comment]
@Jerakin this does not work with version 1.2.136. All it does is focus on the already running instance.

**Comment by Jerakin** [comment]
@davidoshiro Are you sure you are supplying the "`--new`" argument? It works for me with the `--new` argument

**Comment by davidoshiro** [comment]
@Jerakin Yes. All it does is refocuses on the open editor. A new instance does not open. Strange that this editor can only view one project at a time. Most other editors will allow you to view the code from multiple projects. Being able to view multiple projects is useful for beginners like me since the framework requires multiple files. Having to reference code from another project is difficult to say the least. IMHO.
